src/main/java/com/example/getnewjoke/util/AppConst.java

```diff
@@ -0,0 +1,9 @@
+package com.example.getnewjoke.util;
+
+public final class AppConst {
+  public static final String URL_API = "https://v2.jokeapi.dev/joke/Any?type=single";
+  public static final String WSL_IP = "172.27.13.237";
+  public static final String TOPIC = "jokes";
+
+  private AppConst() {}

```

Comment:
no need in this file at all, use application.yaml

Reply:
Done